pax-chassis-web/interface/src/events/listeners.ts

```diff
@@ -104,7 +104,7 @@ export function setupEventListeners(chassis: PaxChassisWeb) {
         if (res.prevent_default) {
             evt.preventDefault();
         }
-    }, {"passive": true, "capture": true});
+    }, {"passive": false, "capture": true});
```

Comment:
why are we making passive false?

Reply:
Without this we can't prevent the default behaviour, resulting in panning with two fingers in the designer triggering navigation back/forward in the website history.

Reply:
ah makes sense